frontend/src/sass/pages/_meet-the-team.scss

```diff
@@ -16,11 +16,28 @@ $team-individual-img-height: 150px;
 
 .team {
 	
+	display: flex;
+	
+	&__first-column {
```

Comment:
I had a look if this can be done in flex http://codepen.io/lili2311/pen/GjbdNz?editors=1100#0 here is an example that retains masonry and keeps the order as it was before. It is a little trickier but it retains the order of people visually sort of. So if you haven't found a way yet check it out.